ui/css/perf.css

```diff
@@ -219,6 +219,47 @@ to { opacity: 0; }
     overflow: hidden;
 }
 
+/* Test Chooser graph */
+
+.modal-body {
+  display: flex;
+  justify-content: space-around;
+  flex-flow: row;
+  align-content: center;
+}
+
+.typeahead {
+  font-size: 16px;
+  width: 250px;
+}
+
+.test-chooser {
+  flex: auto;
+  width: 280px;
+}
+
+.btn-group-vertical {
+  flex: none;
+  position: relative;
+  top: 245px;
+  padding: 3px;
+}
+
+#move-left {
```

Comment:
Let's rename this to `#select-test`, to go along with the proposed naming change below.